eZ/Publish/Core/IO/IOMetadataHandler/Flysystem.php

```diff
@@ -64,7 +65,11 @@ public function load($spiBinaryFileId)
 
     public function exists($spiBinaryFileId)
     {
-        return $this->filesystem->has($spiBinaryFileId);
+        try {
+            return $this->filesystem->has($spiBinaryFileId);
+        } catch (CorruptedPathDetected $e) {
+            return false;
```

Comment:
Looks like something that should be logged.

Reply:
@barw4 is on vacation - he'll need to address this when he's back, as a follow-up. Given we're past QA, I'm not going to introduce risk of changes right now. You're right, but it's not critical.

Reply:
Resolved on a merge up when taking care of other issues https://github.com/ibexa/core/pull/326